__tests__/app.test.js

```diff
@@ -484,4 +484,42 @@ describe.only("GET /api/articles with sorting queries", () => {
       expect(msg).toBe("Not available")
     })
   })
-})
\ No newline at end of file
+})
+describe.only("GET /api/articles topic query", () => {
+test("200: retrieves articles belonging to a specific topic value", () => {
+  return request(app)
+  .get("/api/articles?topic=cats")
+  .expect(200)
+  .then(({body}) => {
+    const {articles} = body
+    expect(articles[0].topic).toBe("cats")
```

Comment:
Rather than just checking the first element in the array, we should loop over all the articles to ensure they all have this topic 👍